src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -106,13 +106,21 @@
 						</template>
 						{{ t('spreed', 'Mark as unread') }}
 					</NcActionButton>
-					<NcActionLink v-if="linkToFile"
-						:href="linkToFile">
-						<template #icon>
-							<File :size="20" />
-						</template>
-						{{ t('spreed', 'Go to file') }}
-					</NcActionLink>
+					<template v-if="isFileShare">
+						<NcActionSeparator />
+						<NcActionLink :href="linkToFile">
+							<template #icon>
+								<File :size="20" />
+							</template>
+							{{ t('spreed', 'Go to file') }}
+						</NcActionLink>
```

Comment:
This wasn't supposed to be shown for guests, no 🤔 ?

Reply:
Is wasn't shown, but I don't think it wasn't supposed to. It was a bug.

There was no link in the menu, but the preview itself is a link anyway.

Reply:
Shared files in public rooms are public shares, so should be correct here:
- as user: `https://nextcloud.local/index.php/f/585`
- as guest: `https://nextcloud.local/index.php/s/A4ERJc3xMKmN88o`